src/main/vue/src/store/actions.ts

```diff
@@ -0,0 +1,5 @@
+export default {
+  setCurrentSound: (context: any, sound: string) => {
```

Comment:
Actions, mutations and getters should be probably grouped in separate files (per feature). I don't know good practices in Wujex but I cannot imagine a big project where all actions are in one big file.

Reply:
Agreed, new concept is underway.